components/admin_console/billing/billing_history.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+type Props = {
+
+};
+
+const BillingHistory: React.FC<Props> = () => {
```

Comment:
You can leave out the `Props` if there aren't any.

```suggestion
const BillingHistory: React.FC = () => {
```

Reply:
Props will likely be added later, but I can remove them now if you want.

Reply:
ahh gotcha, seeing that this is WIP, then it's all good 👍